SMSService/src/lambdas/sendSMS/sendSMS.js

```diff
@@ -0,0 +1,49 @@
+import AWS from "aws-sdk";
+import logger from "../utils/logger.js";
+import { handleError } from "../utils/handleError.js";
+import { checkMessage } from "./utils/checkMessage.js";
+import { checkPhoneNumber } from "./utils/checkPhoneNumber.js";
+import { sendMessage } from "./utils/sendMessage.js";
+
+const sqs = new AWS.SQS();
+
+export const sendSMS = async (event) => {
```

Comment:
Small thing, potentially personal preference. This function name is slightly misleading for what it does. It actually validates the message, the phone number and then if valid sends the SMS, or throws an error. 

It'll be clearer to the reader if you validate, then send if valid.